### ISSUE TITLE
[Bug]: Text line flickering due to line break when chaning CFG slider (+possible CSS fix?)

### ISSUE THREAD

**Issue by nahojjjen** [hlky/stable-diffusion-webui#465]
### What happened?

When dragging the Classifier Free Guidance slider, the text jumps around a lot because the numbers shift between having a decimal and not.

I don't know if the framework limits us from defining the CSS for the slider text, but if we can define the CSS, it can be fixed by adding :
min-width: 5ch;
To the slider text css div. Above css makes the text width always take at least 5 characters width. I tested it by manually changing it in the browser with inspect-element.

Sorry if this is a duplicate issue. Thanks for the great work!

### Version

0.0.1 (Default)

### What browsers are you seeing the problem on?

Firefox, Chrome

### Where are you running the webui?

Windows

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by codedealer** [comment]
Thought this was fixed long time ago? Strange.

**Comment by nahojjjen** [comment]
I redownloaded and re-installed everything yesterday, and Ive pulled the webui repo and replaced the script and frontend files earlier today. I hope I haven't messed up my local install. Can someone else reproduce the issue? 

Here's a screenshot of how it looks for me:
![bild](https://user-images.githubusercontent.com/8545451/188008030-cdad1415-e398-4ebd-a35c-d2e6003f6c5c.png)
The "the" word wraps to the next line if I have a decimal number:
![bild](https://user-images.githubusercontent.com/8545451/188008146-b5df81aa-6372-479d-ba08-1a2b2d2a9294.png)
If I'm on a narrower window (for example if it's snapped to the left of my screen) the text rapidly flickers between 3&4 lines:

![bild](https://user-images.githubusercontent.com/8545451/188008465-102c01e8-077b-40a0-b4fd-7310f285d3bc.png)
![bild](https://user-images.githubusercontent.com/8545451/188008494-a2bcf15b-c9d7-42e7-923f-743a206b20ea.png)

**Comment by codedealer** [comment]
Yep, when I open the window to the full screen it persists. Guess this fix got lost in all the redesign, good catch!

**Comment by codedealer** [comment]
making the width of fixed value should do it, `5ch` is a magic number of course and it can break (overflow) again if in the future there'll be values that are longer than 5 characters long but yes, for now it should be good.

In fact I have a Mandela effect of remembering making this change and exactly `5ch` as well. Maybe I did it in my private fork lol
